### PR TITLE
Create Base module for Tests

### LINKED ISSUE #674: Relocate tests to base Tests directory
## Description
As a contributor,
I need to have all tests together in their own directory in the project root,
so that I can easily look up other tests for reference and write and add tests myself.

## Mocks
<None>

### Update [Required]
- [ ] Create Tests directory
- [ ] Move existing tests to said directory

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3-4 hours

### PR DESCRIPTION
**Pull request by Monal5031**
# Description
This PR aims to create a base module for Tests in root of project where all the tests will be written.

Fixes #674 

# Type of Change:

- Code
- Quality Assurance

**Code/Quality Assurance Only**

- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
Run the following command in root of project
```
python vms/manage.py test Tests/
```

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: Tests/administrator/test_working.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        Delete this function after the first test for this
+        Class is added.
+        """
+        self.driver.get("http://www.python.org")
+        self.assertIn('Python', self.driver.title)
+        element = self.driver.find_element_by_name('q')
+        element.clear()
+        element.send_keys('pycon')
+        element.send_keys(Keys.RETURN)
+        self.assertNotIn('No results found.', self.driver.page_source)
```
Comment (abhishekarya286) [inline Tests/administrator/test_working.py]: No new line at the end in all the files.

Comment (ayushgarg1804) [reply Tests/administrator/test_working.py]: Yes it's recommended by pep8, due to consistency in every line being terminated by '\n'.

Comment (Monal5031) [reply Tests/administrator/test_working.py]: Yes I follow that rule, here is the screenshot of the file in pycharm which shows that there is a newline at end.
newline

Comment (naman1901) [reply Tests/administrator/test_working.py]: Can you check if some other IDE/editor shows the newline too? Something like Sublime, VSCode, etc.?

**Comment by naman1901** [comment]
I'd like to merge this pull request on a priority basis, as another PR is blocked due to this. Can someone in the VMS team review  or should  I go ahead and merge? @ayushgarg1804

File: Tests/administrator/test_working.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Third Party Imports
```
Comment (ayushgarg1804) [inline Tests/administrator/test_working.py]: I find these comments redundant and making below imports harder to read. Are they even required?

File: Tests/administrator/test_working.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+from selenium import webdriver
+
+# Django imports
+from django.contrib.staticfiles.testing import LiveServerTestCase
+
+
```
Comment (ayushgarg1804) [inline Tests/administrator/test_working.py]: Extra line

File: Tests/administrator/test_working.py (hunk 1 above)
Comment (Monal5031) [reply Tests/administrator/test_working.py]: Okay I'll check it, Thanks.

File: Tests/administrator/test_working.py (hunk 2 above)
Comment (Monal5031) [reply Tests/administrator/test_working.py]: Yes, they will be required while adding the tests in future but we can remove it for now.

Comment (Monal5031) [reply Tests/administrator/test_working.py]: I can remove them, but it is a good practice to separate the type of imports namely (third-party, local-project, django)

**Comment by tapasweni-pathak** [comment]
Hey you all, I'm not able to keep up to VMS PR's and issue queue, but this should never have been merged. 

What is the use of adding base test? In future this would further require cleanup and lowering the percentage and confusion for contributors. What does these tests test? Can anyone please link me to the use of having such test in a project like VMS. It is great to have thoughts from people having different level of understanding of Django, Tests, but PR's should be merged after getting a review from someone who is a bit more known to all these. 

cc @systers/maintainers-vms @systers/admins.

**Comment by tonythomas01** [comment]
`python vms/manage.py test Tests/` seems like an out of the ordinary idea. Is the specific reasons to do this documented somewhere ?

**Comment by Monal5031** [comment]
@tapasweni-pathak This was in my gsoc proposal (infrastructure/automation for web) that I'll start by creating a separate dir for Tests where in only tests corresponding to project will be maintained not a different repo itself.
Yes it will lower the coverage for sometime but its part of my gsoc  to enhance it. This creation of structure followed by integrating selenium on travis (which it currently fails) will allow @anjali-dhanuka to write tests for her enhancements from start of gsoc itself.
Also for now to check if the new structure is working I have added dummy tests and did not move the original tests yet but I will as I progress through my gsoc and Sure I agree it'd great o have review from the maintainers on PRs like this.

@tonythomas01 since I'm relocating tests so the original command to run test wont work we have to give the new path where they are located and I have documented this in the next PR like how to run tests for this new structure.

**Comment by tonythomas01** [comment]
tbh if this move is not critical to your GSoC project, I would recommend leaving it where it is though (and inside apps/tests/), rather than ending up in some issues later.

**Comment by Monal5031** [comment]
@tonythomas01 @tapasweni-pathak Its not much needed change so I've reverted this change in my selenium integration PR.

**Pull request merged**